File: PRAK601/PRAK601-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int x, y, i, j, matriks[20][20];

    scanf("%d %d", &x, &y);

    for (i = 0; i < x; i++)
    {
        for (j = 0; j < y; j++)
        {
            scanf("%d", &matriks[i][j]);
        }
    }

    for (i = 0; i < x; i++)
    {
        for (j = 0; j < y; j++)
        {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
}

File: PRAK601/PRAK601-2210817310013-Ryan Muhammad Irfan.py
x, y = map(int, input("").split())
number = list(map(int, input("").split()))
matriks = []
A = 0

for i in range(x):
    matriks.append(number[A: A + y])
    A = A + y

for x in (matriks):
    for number in (x):
        print(number, end=" ")
    print()
